lib/gateways/sessions.rb

```diff
@@ -3,15 +3,18 @@ class Sessions
     MAXIMUM_RESULTS_COUNT = 500
 
     def initialize(ips:)
-      @ips = ips
+      @organisation_ips = ips
```

Comment:
Just curious what the concern was with this variable renaming?

Reply:
I think it's not clear to the reader / consumer of the use case what these `ips` are, because there are two concepts named `ips`.

Happily taking suggestions on names to differentiate _ips that I'm allowed to show because they belong to the organisation_ and _ips that I've requested to filter the search by_.

(I actually wanted to rename the named parameter too, but that broke the interface so I left it 'til later.)

Reply:
Ah right yes I see what you mean